sdcm/utils/raft/common.py

```diff
@@ -44,6 +50,27 @@ def validate_raft_on_nodes(nodes: list[BaseNode]) -> None:
     LOGGER.debug("Raft is ready!")
 
 
+def get_topology_coordinator_node(cluster: BaseScyllaCluster) -> BaseNode:
+    active_nodes = cluster.get_nodes_up_and_normal()
```

Comment:
maybe this function should be guarded with ` self.target_node.raft.is_consistent_topology_changes_enabled` ?

to make sure it's not getting called when raft topology is disabled ?

Reply:
added